### PR TITLE
📝 Add Console steps for source integrations

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

The docs for source integrations incorrectly stated that the CLI was necessary.

## What's changed

- Added steps for Console.
- Tried to organize shortcodes a bit. 
- Took out statements about CLI being necessary.
- Did not validate or standardize the entire pages. Just some small things in passing.

### CODE REVIEW

**Review by AnouckColson — CHANGES_REQUESTED** [review]
Very clear content 👍 .
Kinda fell into the rabbit hole of formatting in the GitHub/GitLab procedures, didn't mean to. Maybe slightly out of scope but easy to fix, so I figured I'd leave my suggestions for you to go over.

File: docs/src/integrations/source/gitlab.md [hunk 1]
```diff
@@ -85,37 +90,51 @@ Optional parameters:
...
 If you see the message `Failed to read or write webhooks`, you need to add a webhook manually:
 
-1. Copy the hook URL shown in the message.
-2. Go to your GitLab repository and click Settings > Webhooks.
-3. Paste the hook URL.
+1. Get the webhook URL by running `platform integration:get`
+2. Copy the `hook_url`.
+3. Go to your GitLab repository and click Settings > Webhooks.
+4. Paste the hook URL.
    In the Triggers section choose Push events, Tag push events and Merge Request events.
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/src/integrations/source/gitlab.md:130]:
```suggestion
   In the **Trigger** section select **Push events**, **Tag push events**, and **Merge request events**.
```

**Comment by CollierCZ** [comment]
There were definitely a lot of style and other issues that I didn't fix because I worried about falling into that hole you mentioned. Definitely good to fix them if it's easy to do so, which it was with your suggestions, so thanks!

File: docs/src/integrations/source/bitbucket.md [hunk 2]
```diff
@@ -66,59 +66,70 @@ Copy the token and make a note of it (temporarily).
...
-* `PLATFORMSH_PROJECT_ID` is the project ID for your Platform.sh project.
-* `BASE_URL`: The base URL of the server installation.
-* `USERNAME`: Your Bitbucket Server username.
-* `TOKEN`: The access token you created for the integration.
-* `REPOSITORY`: The repository  (e.g. 'owner/repository').
-
-## Validate the integration
+{{< codetabs >}}
+---
+title:Using the CLI
```
Comment (AnouckColson) [inline docs/src/integrations/source/bitbucket.md:74]: 🛠️ 

```suggestion
title=Using the CLI
```

File: docs/src/integrations/source/bitbucket.md [hunk 3]
```diff
@@ -66,59 +66,70 @@ Copy the token and make a note of it (temporarily).
...
 
-```bash
-platform help integration:update
-```
+Note that the `--prune-branches` option depends on `--fetch-branches` being enabled.
+If `--fetch-branches` is disabled, `--prune-branches` is automatically set to false, even if specifically set to true.
+
+<--->
+---
+title:In the Console
```
Comment (AnouckColson) [inline docs/src/integrations/source/bitbucket.md:106]: 🛠️ 

```suggestion
title=In the Console
```

File: docs/src/integrations/source/github.md [hunk 4]
```diff
@@ -32,33 +32,39 @@ Copy the token and make a note of it (temporarily).
...
 
-Note that only the project owner can manage integrations.
+To enable the integration, use either the [CLI](../../administration/cli/_index.md)
+or the [Console](../../administration/web/_index.md).
 
-Open a terminal window (you need to have the Platform.sh CLI installed).
-Enable the GitHub integration as follows:
+{{< codetabs >}}
+---
+title:Using the CLI
```
Comment (AnouckColson) [inline docs/src/integrations/source/github.md:42]: 🛠️ 

```suggestion
title=Using the CLI
```

File: docs/src/integrations/source/github.md [hunk 5]
```diff
@@ -73,36 +79,55 @@ Optional parameters:
...
-
-If you see the message `Failed to read or write webhooks`, you need to add a webhook manually:
-
-1. Copy the hook URL shown in the message.
-2. Go to your GitHub repository and click Settings, select the Webhooks and Services tab, and click Add webhook.
-3. Paste the hook URL, choose `application/json` for the content type,
-   choose "Send me everything" for the events you want to receive, and click Add webhook.
+<--->
+---
+title:In the Console
```
Comment (AnouckColson) [inline docs/src/integrations/source/github.md:87]: 🛠️ 

```suggestion
title=In the Console
```

File: docs/src/integrations/source/gitlab.md [hunk 6]
```diff
@@ -40,31 +40,36 @@ but that's attached to a specific user rather than the project as a whole.
...
 
-Note that only `project owner` or `project admin` can manage the integrations.
+To enable the integration, use either the [CLI](../../administration/cli/_index.md)
+or the [Console](../../administration/web/_index.md).
 
-Open a terminal window (you need to have the Platform.sh CLI installed).
-Enable the GitLab integration as follows:
+{{< codetabs >}}
+---
+title:Using the CLI
```
Comment (AnouckColson) [inline docs/src/integrations/source/gitlab.md:50]: 🛠️ 

```suggestion
title=Using the CLI
```

File: docs/src/integrations/source/gitlab.md [hunk 7]
```diff
@@ -85,37 +90,51 @@ Optional parameters:
 Note that the `--prune-branches` option depends on `--fetch-branches` being enabled.
 If `--fetch-branches` is disabled, `--prune-branches` is automatically be set to false, even if specifically set to true.
 
-### 3. Add the webhook
+<--->
+---
+title:In the Console
```
Comment (AnouckColson) [inline docs/src/integrations/source/gitlab.md:95]: 🛠️ 

```suggestion
title=In the Console
```

**Review by AnouckColson — CHANGES_REQUESTED** [review]
Sorry I only just realized there were other instances of the syntax being wrong for the display of Console vs CLI tabs in the file. Should all be fixed after this last review.

**Review by AnouckColson — APPROVED** [review]
🚀 Ready to go!

**Pull request merged**